charts/vault/values/plugin-secrets-kubernetes-reader.yaml

```diff
@@ -0,0 +1,34 @@
+server:
```

Comment:
Not related to plugin, like other fields here. 

Consider making this strictly for the things required to setup the plugin so it can be merged later with production deployment values. This is the file in this PR that can be useful beyond just our testing.